examples/carrier accounts/update_carrier_account.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'easypost'
+
+EasyPost.api_key = ENV['EASYPOST_TEST_API_KEY']
+
+ca = EasyPost::CarrierAccount.retrieve('ca_...')
+
+ca.description = 'FL Location UPS Account'
+
```

Comment:
Nit: no need for a line break here, we want to group like things together (so updating the description and credentials could be one after another.